src/types/commands.ts

```diff
@@ -502,6 +502,7 @@ export interface UpdateLocaleCommand {
 // ------------------------------------------------
 export interface CopyCommand {
   type: "COPY";
+  copyTarget?: Zone[];
```

Comment:
I do'nt agree. I changed the "target" argument from cut to cutTarget, because "target" is not a keyword in commands and should always be a array of zones (not optional). We can argue whether or not we should allow optional, but it just makes the typing/generics functions easier.

And in the long/medium term we should make this argument disappear in CUT, the only reason it's needed is because we use CUT/PASTE to move columns, but we should really not and either have separate command or use the ClipboardState directly like Alexis.

Reply:
@hokolomopo Apologies, you are correct. `target` is reserved and should stay so.

The current sstrategy (470bec6b6005541da7d63fd44e9d3765393e6c7d) seems to have overlooked that issue by providing a custom command :)